src/backend/JasmineGraphBackendProtocol.h

```diff
@@ -34,10 +34,16 @@ extern const string LOCAL_TO_WORLD_AUTHFLOW_FOR_PARTITION;
 extern const string WORLD_TO_LOCAL_FLOW_FROMIDS;    //This command gets all the vertices connected with the external graph and their corresponding fromIDs
 extern const string DONE;
 extern const string PARTITIONS_ON_HOST;     //The command to get the list of partitions on particular host
-extern const string RECORD_PERF_STATS; // Command to record the performance statistics in database
+extern const string ACKNOWLEGE_MASTER; // Command to record the performance statistics in database
```

Comment:
Lets remove this comment. It has different meaning than the command `ACKNOWLEGE_MASTER`

Reply:
Will change the comment

Reply:
Ok